packages/stream_video_flutter/lib/src/call_participants/screen_share_call_participants_content.dart

```diff
@@ -146,11 +153,18 @@ class ScreenShareContent extends StatelessWidget {
             }
 
             return InteractiveViewer(
+              maxScale: 4,
+              onInteractionUpdate: (details) {
+                setState(() {
+                  _currentScale = details.scale;
+                });
+              },
               child: StreamVideoRenderer(
-                call: call,
-                participant: participant,
+                call: widget.call,
+                participant: widget.participant,
                 videoFit: VideoFit.contain,
                 videoTrackType: SfuTrackType.screenShare,
+                persistTrackIfNotVisible: _currentScale > 1.0,
```

Comment:
I wonder if this adds way to many rebuilds. Because we're setting the state every time you zoom a bit.

Can't we just always set this to true always, or only update the state if the zoom starts/stops? For example:
```dart
              onInteractionUpdate: (details) {
                final isZoomed = details.scale > 1.0;
                if(_isZoomed != isZoomed){
                  setState(() {
                    _isZoomed = isZoomed;
                  });
                },
              },
```